src/api/firebase.js

```diff
@@ -20,7 +22,7 @@ const firebaseConfig = {
 };
 
 const app = initializeApp(firebaseConfig);
-const db = getFirestore(app);
+export const db = getFirestore(app);
```

Comment:
good call exporting db from here!

Reply:
This was actually from a previous iteration! We originally built all logic in `Home.jsx` and exported what we needed - but we have since moved it over to `Firebase.js` so I don't think this needs to be exported anymore! I have deleted it.

Reply:
Good work on the refactor! db should always remain in the firebase.js file and never need to be exported or imported. We want to isolate it to that file and use it with the api calls that need it and then import export those client-side Create, Read, Update Destroy api functions.